Microsoft.Toolkit.Uwp.UI/Helpers/DesignTimeHelpers.cs

```diff
@@ -13,22 +13,31 @@
 using System;
 using System.Reflection;
 
-namespace Microsoft.Toolkit.Uwp.UI.Controls
+namespace Microsoft.Toolkit.Uwp.UI
 {
     /// <summary>
-    /// Internal class used to provide helpers for controls
+    /// Class used to provide helpers for design time
     /// </summary>
-    internal static partial class ControlHelpers
+    public static class DesignTimeHelpers
```

Comment:
Why the change in visibility? Seems unrelated

Reply:
It's unrelated, will change back

Reply:
Actually now that I understand you moved it out to a different assembly for reuse, I guess you have to make it public